app/assets/stylesheets/templates/_ctc-home.scss

```diff
@@ -18,10 +18,34 @@
     padding-bottom: 0;
   }
 
-  .slab--why {
+  .slab--affiliations {
     background-color: $color-grey-light;
   }
 
+  .affiliations {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+
+    .grid__item {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
+    .grid__item.text {
+      width: 200px;
```

Comment:
what happens if you pull this up to like 225? I noticed that the text under the first image has two lines with only one word each on it which looks a little weird. If each text section had a bit more width, we might be able to avoid orphaned words.